e2e/testcases/sync_ordering_test.go

```diff
@@ -442,18 +442,14 @@ func TestDependencyWithReconciliation(t *testing.T) {
 	nt.WaitForRepoSyncs()
 
 	var pod1, pod2 *corev1.Pod
-	_, err := nomostest.Retry(nt.DefaultWaitTimeout, func() error {
+	_, err := nomostest.Retry(30*time.Second, func() error {
```

Comment:
What's the reason for `Retry` rather than switching to watch for these?

Reply:
These do some weird logic that uses the Validate to get the Pod objects and use them for additional validation. So I just left them alone. 

It’s possible there’s a way to clean them up but it wasn’t immediately obvious. They use one object to check another object, so a normal Predicate wouldn’t work unless you do them in serial.

Reply:
Ack